dependencies/Boost/gcc_5_no_cxx11.patch

```diff
@@ -0,0 +1,37 @@
+From eec808554936ae068b23df07ab54d4dc6302a695 Mon Sep 17 00:00:00 2001
```

Comment:
One question: why was this necessary? gcc 5.2 supports C++11/14/17

And to answer your question, I'd strongly recommend against applying where not needed because it could have a significant impact on compile times (variadic templates are much faster than the alternative).